recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+MACOSX_SDK_VERSION:        # [osx]
+  - "11.3"                 # [osx]
```

Comment:
Can you document in a comment why this is the used SDK version?

Reply:
Just changed to minimal and added a comment